website/src/index.css

```
@@ -16,3 +16,315 @@ body {
+  }
+}
+
+/* Easing curves — natural deceleration only */
+:root {
+  --ease-out-quart: cubic-bezier(0.25, 1, 0.5, 1);
+  --ease-out-quint: cubic-bezier(0.22, 1, 0.36, 1);
+  --ease-out-expo: cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+/* Fade + slide up entrance */
+@keyframes fadeSlideUp {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeSlideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)